.gitignore

```diff
@@ -101,4 +103,4 @@ venv.bak/
 /site
 
 # mypy
-.mypy_cache/
+.mypy_cache/
```

Comment:
please don't remove the final newline in a file

it seems like your editor is not picking up the `.editorconfig` file (see rule [here](https://github.com/maykinmedia/maykin-auth-matrix/blob/main/.editorconfig#L7))